pwiz_tools/Skyline/Model/Export.cs

```diff
@@ -4125,6 +4125,33 @@ protected override void WriteHeaders(TextWriter writer)
             }
             writer.WriteLine();
         }
+
+        // Hack to replace modification brackets with parentheses.
+        // MassLynx or VerifyESkylineLibrary.dll has some problem with multiple sets of brackets that causes
+        // an incomplete method to be generated.
+        public static string FormatMods(string modSeq)
+        {
+            if (!modSeq.Contains('['))
+                return modSeq;
+
+            var newSeq = new StringBuilder(modSeq.Length);

```

Comment:
I wonder if this is really necessary. It seems like replacing this entire function with:

return modeSeq.Replace('[', '(').Replace(']', ')');

Would be just fine given all the other work that is happening. I can't imagine the double allocation ever showing up in a profiler as important.